.github/workflows/all-pr-tests.yml

```diff
@@ -47,3 +47,23 @@ jobs:
           php-version: '${{ matrix.php }}'
           wordpress-version: '${{ matrix.wordpress }}'
           skip-wordpress-install: 'true'
+  # This required job ensures that all PR checks have passed before merging.
```

Comment:
This one is new to me. What's the difference between this and the branch protection rules managed in GitHub?